ldms/python/ldmsd/ldmsd_yaml_parser

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+import os, sys
+import yaml
+import errno
+import argparse
+from ldmsd.parser_util import *
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="LDMS Monitoring Cluster Configuration")
+    parser.add_argument("--ldms_config", metavar="FILE", required=True,
+                        help="The ldmsd load balance domain configuration file. "
+                        "This will not start the maestro "
+                        "load balancer")
+    parser.add_argument("--generate-config-path", metavar="STRING", required=False,
+                        default=False)
+    parser.add_argument("--daemon_name", metavar="STRING", required=False,
+                        default=False,
+                        help="Daemon name to generate configuration from YAML file")
+    parser.add_argument("--version", metavar="VERSION",
+                        help="The OVIS version for the output syntax (4 or 5), default is 4",
+                        default=4)
+    parser.add_argument("--debug", action="store_true",
+                        help="Enable debug information")
+    args = parser.parse_args()
+    if not args.debug:
+        import sys
+        sys.tracebacklimit=0
```

Comment:
We should _not_ do this. The backtrace is our only way to reasonably debug things when there are bugs in the code.

Reply:
I agree with you, and that's how it was originally. However, there were some in the community that wanted the option to not see a debug error message, hence the "--debug" option.